<launch>
    
    <!-- Set the joystick input -->
    <arg name="robot_name" default="trina2"/>


    <!-- falcon dirver -->
    <node respawn="true" pkg="ros_falcon" type="driver" name="falcon_driver" output="screen">
    </node>

    <!-- falcon interface node -->
    <node name="falcon_interface" pkg="trina2_control" type="falcon_interface" output="screen"> 
        <remap from="/set_left_vel" to="$(arg robot_name)/set_left_vel" />
        <remap from="/set_left_eef" to="$(arg robot_name)/set_left_eef" />
    </node>

</launch>